.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-## Pull Request Guidelines
+<!--
+Checklist for submitting a Pull Request
+
+Please remove this comment block before submitting.
 
 1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
+3. If you want this PR to automatically close an issue when it is merged,
+   add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
```

Comment:
I cede any control I had to Mr. McMahon

On Tue, May 17, 2016 at 7:17 PM, Xiangpeng Zhao <notifications@github.com>
wrote:

> In .github/PULL_REQUEST_TEMPLATE.md
> <https://github.com/kubernetes/kubernetes/pull/25717#discussion_r63636299>
> :
>
> >
> >  1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
> > -1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
> > -1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
> > +2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
> > +3. If you want this PR to automatically close an issue when it is merged,
> > +   add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
>
> What if someone accidentally type an unintended issue number? Thanks!
> @asalkeld <https://github.com/asalkeld>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25717/files/448b1d2fde7bee29e49ffbd882e3a01e0b40971d#r63636299>
>